pilot/pkg/util/network/ip.go

```
@@ -193,6 +202,22 @@ func AllIPv4(ipAddrs []string) bool {
 	return true
 }
 
+// CheckIPFamilyTypeForFirstIPs checks the ip family type for the first ip addresses
+func CheckIPFamilyTypeForFirstIPs(ipAddrs []string) IPFamilyType {
+	if len(ipAddrs) == 0 {
+		return UNKNOWN
+	}
```

if this happens it should return an error that should be propagated up

---

Hi @jacob-delgado okay, done.